api/room/mutations.js

```diff
@@ -18,12 +21,21 @@ export default {
       return { success: false, message: "Failed to add room" };
     }
   },
-  updateRoom: async (_, { id, users }) => {
-    if (updateRoom(id, users)) {
-      return { success: true, message: "Room updated" };
-    } else {
-      return { success: false, message: "Failed to update room" };
+  updateRoom: async (_, { id, users, isShown }) => {
+    try {
+      if (updateRoom(id, users, isShown)) {
+        const room = getRoomByID(id)
+        pubsub.publish("ROOM_UPDATED", {
+          roomUpdated: room
+        });
+        return { success: true, message: "Room updated" };
+      } else {
+        return { success: false, message: "Failed to update room" };
+      }
+    } catch (error) {
+      console.log('update room error', error);
```

Comment:
This can return
return { success: false, message: "Failed to update room" };

instead of logging